### PR TITLE
Pulling images

### PR DESCRIPTION
**Pull request by saibaskaran**

### CODE REVIEW

File: layout.html [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+                    <img src="images/swami.jpg" height="150" width="relative">
+                </td>
+            </tr>
+        </table>
+    </header>
+    <nav class="nav-menu">
+        <ul>
+            <li><h1><a href="#home">Home</a></h1></li>
+            <li><h1><a href="#bhajans">Bhajans</a></h1></li>
+            <li><h1><a href="#specialevents">Special Events</a></h1></li>
```
Comment (Jagannath-Ramesh) [inline layout.html]: add section for specialevents so that it navigates to that section when clicked

**Comment by Jagannath-Ramesh** [comment]
Squash before merging. Add good description after squashing.

👍 after

File: layout.html (hunk 1 above)
Comment (saibaskaran) [reply layout.html]: good catch!

Comment (saibaskaran) [reply layout.html]: done!

**Comment by saibaskaran** [comment]
Done with the changes. Merging pull request

**Pull request merged**